File: java/Shirt.java
package LP2.java;

public class Shirt extends Clothing{
    // varible private
    private String color;
    private String sleeve_type;

    // constructor
    public Shirt(){
        this.color = "";
        this.sleeve_type = "";
    }

    public Shirt(String sleeve_type, String color) {
        this.color = color;
        this.sleeve_type = sleeve_type;
    }
    
    // setter dan getter
    public String getColor(){
        return this.color;
    }

    public void setColor(String color){
        this.color = color;
    }

    public String getSleeveType(){
        return this.sleeve_type;
    }

    public void setSleeveType(String sleeve_type){
        this.sleeve_type = sleeve_type;
    }
}

File: php/Program/Shirt.php
<?php

require('Clothing.php');

class Shirt extends Clothing
{
     //private atribut
     private $color;
     private $sleveeType;
     
     //constructor
     public function __construct($color = '', $sleveeType = '')
     {
         $this->color = $color;
         $this->sleveeType = $sleveeType;
     }

     public function setColor($color)
     {
         $this->color = $color;
     }

     public function getColor()
     {
         return $this->color;
     }

     public function setsleveeType($sleveeType)
     {
         $this->sleveeType = $sleveeType;
     }

     public function getsleveeType()
     {
         return $this->sleveeType;
     }
}
?>